### PR TITLE
documentation updates for v2.4.1

### PR DESCRIPTION
**Pull request by kishorj**
### Description
Added live docs walkthrough to create RBAC for secret resources.

### Tests
- ran make docs-preview and verified the live docs rendered properly

### Checklist
- [x] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: helm/aws-load-balancer-controller/README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ If you are setting `serviceMonitor.enabled: true` you need to have installed the
 
 ## Installing the Chart
 **Note**: You need to uninstall aws-alb-ingress-controller. Please refer to the [upgrade](#Upgrade) section below before you proceed.
+**Note**: Starting chart version 1.4.1, you need to explicitly set clusterSecretsPermissions.allowAllSecrets to true to grant the controller permission to access all secrets for OIDC feature. We recommend configuring access to individual secrets resource separately.
```
Comment (M00nF1sh) [inline helm/aws-load-balancer-controller/README.md:89]: maybe link to "docs/examples/secrets_access.md"?

File: mkdocs.yml [hunk 2]
```diff
@@ -33,6 +33,7 @@ nav:
   - Examples:
     - EchoServer: examples/echo_server.md
     - gRPCServer: examples/grpc_server.md
+    - RBAC for Secrets: examples/secrets_access.md
```
Comment (M00nF1sh) [inline mkdocs.yml:36]: i think `RBAC to access OIDC Secret` or `RBAC to access Secret` might be better

File: helm/aws-load-balancer-controller/README.md (hunk 1 above)
Comment (kishorj) [reply helm/aws-load-balancer-controller/README.md:89]: done

File: mkdocs.yml (hunk 2 above)
Comment (kishorj) [reply mkdocs.yml:36]: done

**Review by M00nF1sh — APPROVED** [review]
/lgtm

**Pull request merged**